### PR TITLE
Fix push notification character counter not working

### PR DESCRIPTION
**Pull request by charludo**
### Short description

Fix the character counting that was broken as a side effect from #1713 

### Proposed changes

- update the query responsible for finding textareas with a character counter

### Side effects

- none 

### Resolved issues

Fixes: #1886 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Review by ulliholtgrave — APPROVED** [review]
Thanks, looks good :+1:

**Review by timoludwig — APPROVED** [review]
Thanks! 👍
Maybe add a changelog entry for this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ UNRELEASED
 2022.11.4
 ---------
 
+* [ [#1886](https://github.com/digitalfabrik/integreat-cms/issues/1886) ] Fix push notification character counter not working
```
Comment (timoludwig) [inline CHANGELOG.md:8]: Could you move this into the unreleased section?
```suggestion
* [ [#1886](https://github.com/digitalfabrik/integreat-cms/issues/1886) ] Fix push notification character counter
```

**Review by timoludwig — CHANGES_REQUESTED** [review]

**Review by timoludwig — APPROVED** [review]
:+1:

**Pull request merged**